pkg/app/master/command/images/model.go

```diff
@@ -0,0 +1,166 @@
+package images
+
+import (
+	"time"
+
+	"github.com/charmbracelet/bubbles/key"
+	"github.com/charmbracelet/lipgloss"
+	"github.com/charmbracelet/lipgloss/table"
+	"github.com/dustin/go-humanize"
+	"github.com/mintoolkit/mint/pkg/app"
+
+	"github.com/mintoolkit/mint/pkg/app/master/command"
+	"github.com/mintoolkit/mint/pkg/app/master/tui/common"
+	"github.com/mintoolkit/mint/pkg/app/master/tui/keys"
+	"github.com/mintoolkit/mint/pkg/crt"
+	"github.com/mintoolkit/mint/pkg/crt/docker/dockerutil"
+
+	tea "github.com/charmbracelet/bubbletea"
+)
+
+// Model represents the state of the TUI.
+type Model struct {
+	table      table.Table
+	width      int
+	height     int
+	standalone bool
+	loading    bool
+}
+
+// Styles - move to `common`
+const (
+	gray      = lipgloss.Color("#737373")
+	lightGray = lipgloss.Color("#d3d3d3")
+	white     = lipgloss.Color("#ffffff")
+)
+
+var (
+	// HeaderStyle is the lipgloss style used for the table headers.
+	HeaderStyle = lipgloss.NewStyle().Foreground(white).Bold(true).Align(lipgloss.Center)
+	// CellStyle is the base lipgloss style used for the table rows.
+	CellStyle = lipgloss.NewStyle().Padding(0, 1).Width(14)
+	// OddRowStyle is the lipgloss style used for odd-numbered table rows.
+	OddRowStyle = CellStyle.Foreground(gray)
+	// EvenRowStyle is the lipgloss style used for even-numbered table rows.
+	EvenRowStyle = CellStyle.Foreground(lightGray)
+	// BorderStyle is the lipgloss style used for the table border.
+	BorderStyle = lipgloss.NewStyle().Foreground(white)
+)
+
+// End styles
+
+func LoadModel() *Model {
```

Comment:
It'll be good to make it `LoadTUI`, so it's more obvious for casual observers

Reply:
Noted and addressed.